test/test_assets_d8/src/project.make

```diff
@@ -1,15 +0,0 @@
-core = 8.x
-api = 2
-
-; Drupal Core
-projects[drupal][version] = "8.0.4"
```

Comment:
Dropping test coverage for Drush Make for D8 implies this should be merged into pre-v1. I'm okay with that.